### DISCUSSION TITLE
「質問」の記録

### DISCUSSION THREAD

**Discussion by robozushi10** [increments/qiita-discussions#201] [要望・アイデア・不具合]
**これまでの「質問」の記録を辿ることはできるのでしょうか？**

地道に「もっと見る」を押して過去の質問を見るのはツライです.
// 検索オプションで絞れるのかは不明です

同じような質問を繰り返さないようにしないと、回答する人もいなくなると思いました.

**質問のトップページ**
https://qiita.com/question-trend

すでに質問をした機能があるようでしたら、申し訳ありません.

**Comment by kskwtnk** [comment]
@robozushi10 
ご意見ありがとうございます！

申し訳ありません、現状「もっと見る」でしか辿れません…… 🙇 

過去のものを辿るにしても、目当ての質問や回答を探すにしても、質問機能の検索まわりが弱いままとなっております
運営としても認識はしているのですが直近の計画やリソース上すぐに対応が難しそう……というのが正直な回答です

ただ、改善する項目としては挙がっていますので、ご了承いただけますと幸いです 🙇

**Comment by wataru86** [comment]
@robozushi10 

お待たせしました！質問検索機能をベータ版としてリリースしました :rocket: 
ぜひお試しください 💁

[リリースノート](https://qiita.com/release-notes/2023-07-18)
[ヘルプ](https://help.qiita.com/ja/articles/qiita-search-options)
